spec/Drivetrain/BatteryManagement.vspec

```diff
@@ -10,25 +10,47 @@
 # Battery management system for EV / Hybrids
 #
 
-- BatteryTemperature:
+- Battery:
+  type: branch
+  description: Battery status
+
+- Battery.Temperature:
   datatype: float
   type: sensor
   unit: celsius
   description: Temperature of the battery pack
 
-- BatteryCapacity:
-  datatype: int8
+- Battery.StateOfCharge:
+  datatype: uint8
   type: sensor
   unit: percent
   min: 0
   max: 100
-  description: Remaining capacity of the batter pack
```

Comment:
Why was it removed? 

Also, it seems there should be a read-only attribute to indicate the nominal battery capacity of this particular car.